corehq/apps/hqwebapp/templates/hqwebapp/key_value_mapping.html

```diff
@@ -22,8 +22,18 @@
                 <a href="#" data-bind="click: $parent.removeItem" class="btn btn-danger">
                     <i class="icon icon-white icon-remove"></i> Delete
                 </a>
-                <span class="help-inline"
-                      data-bind='visible: $parent.isItemDuplicated(ko.utils.unwrapObservable(key))'>Key is duplicated.</span>
+                <div class="help-inline"
+                      data-bind='visible: $parent.isItemDuplicated(ko.utils.unwrapObservable(key))
+                '>
+                    <div>
+                        Key is duplicated.
+                    </div>
+                    <div data-bind="
+                        visible: key().indexOf(' ') > -1 || key().indexOf('_') > -1
+                    ">
+                        (Note that space and underscore are considered equal in ID mapping keys)
+                    </div>
+                </div>
```

Comment:
See comment above, but I think it's not great to introduce this quirk, because then invariably people will be sloppy and come to rely on the behavior, and then we will never be allowed to change it in the future and we'll be stuck with this arbitrary restraint!